CombatTracker.js

```diff
@@ -261,6 +261,9 @@ function init_combat_tracker(){
 				$("[data-current] button.openSheetCombatButton").click();
 			}
 			let newTarget=$("#combat_area tr[data-current=1]").attr('data-target');
+			window.TOKEN_OBJECTS[newTarget].highlight(!get_avtt_setting_value("highlightScrolling"));
+			window.MB.sendMessage("custom/myVTT/highlight", {id: newTarget});

```

Comment:
I think we should consider putting the first .highlight() in ct_load further down in this file.

Can remove the sendmessage if we do that just save on server messaging really.